manifests/server.pp

```diff
@@ -31,32 +31,32 @@
 #
 # $manage_user::                       Whether to manage the puppet user resource
 #
-# $user::                              Name of the puppetmaster user.
+# $user::                              Name of the puppetserver user.
 #
-# $group::                             Name of the puppetmaster group.
+# $group::                             Name of the puppetserver group.
 #
 # $dir::                               Puppet configuration directory
 #
-# $ip::                                Bind ip address of the puppetmaster
+# $ip::                                Bind ip address of the puppetserver
 #
-# $port::                              Puppet master port
+# $port::                              Puppet server port
 #
 # $ca::                                Provide puppet CA
 #
 # $ca_crl_filepath::                   Path to ca_crl file
 #
-# $ca_crl_sync::                       Sync the puppet ca crl to compile masters. Requires compile masters to
-#                                      be agents of the CA master (MOM) defaults to false
+# $ca_crl_sync::                       Sync the puppet ca crl to compilers. Requires compilers to
+#                                      be agents of the CA server (MOM) defaults to false
```

Comment:
MOM here refers to Master of Masters. I'm not sure what the new term for that is.

Reply:
The 2019.8 docs state: 
> "We've unified infrastructure terminology across all installation types. We now call compile masters compilers to reflect their role: compiling catalogs. Similarly, we call the master a master, whether or not your installation includes compilers. In high availability installations, the node that replicates data from the master is simply a replica. R.I.P. MoM, master of masters, and primary master replica."

but tbh i'm not sure i understand what is supposed to replace MoM? simply Master? how does that fit with the change from master to server everywhere?

Reply:
@binford2k not sure who could answer this. Would you mind taking a look or redirecting us to some source that documents it?

Reply:
Apologies for missing this earlier. We're using "primary server" where we used to use MoM. So in HA you then have replicas of the primary. Compilers remain the same.